tempo_embeddings/embeddings/model.py

```diff
@@ -298,10 +308,10 @@ def mean_pooling(model_output, attention_mask):
         )
 
     @torch.no_grad()
-    def _passage_embeddings(
-        self, passages: list[Passage], store_tokenizations: bool
+    def embed_passages(
+        self, passages: list[Passage], store_tokenizations: bool, skip_batching: bool = False
     ) -> Iterable[torch.Tensor]:
-        for batch in self._batches(passages):
+        for batch in self._batches(passages, skip_batching=skip_batching):
```

Comment:
Still based on the above suggestion, this line would change to:

```suggestion
        for batch in corpus.passage_batches(batch_size):
```